src/base/bittorrent/torrentimpl.cpp

```diff
@@ -801,7 +801,8 @@ QString TorrentImpl::filePath(const int index) const
 QString TorrentImpl::actualFilePath(const int index) const
 {
     const auto nativeIndex = m_torrentInfo.nativeIndexes().at(index);
-    return QString::fromStdString(m_nativeHandle.torrent_file()->files().file_path(nativeIndex));
+    const std::string filePath = m_nativeHandle.torrent_file()->files().file_path(nativeIndex);
+    return Utils::Fs::toUniformPath(QString::fromStdString(filePath));
```

Comment:
I actually expect this function to return path with native separators.
And your commit message does not provide any insight why this change was made 👎

Reply:
filePath() returns a uniform path, this shouldn't be necessary after path overhaul so I can remove this commit.

Reply:
>I actually expect this function to return path with native separators.

This makes the interface of `Torrent` inconsistent, so I take it as a bug.
BUT I am already looking forward to the moment when such problems will be a thing of the past...

Reply:
OK for now. I'll think of it in the overhaul PR.